react/Arrow.js

```diff
@@ -0,0 +1,28 @@
+import React, { Component } from 'react'
+import PropTypes from 'prop-types'
+
+class Arrow extends Component {
+  render() {
+    const { className, style, onClick, color } = this.props
+    return (
+      <div
+        className={className}
+        style={{ ...style, color: color }}
+        onClick={onClick}
+      />
+    )
+  }
+}
+
+Arrow.propTypes = {
```

Comment:
Lembra de colocar o `required` quando for required.

Reply:
Na verdade esse onClick não é obrigatório, pois ele é passado dinamicamente pelas props depois (o react-slick que faz isso)
Deixei o 'color' como obrigatório

Reply:
Okok